doc/_static/mpl.css

```diff
@@ -19,7 +19,7 @@ body {
 }
 
 a {
-    color: #CA7900;
+    color: #CA7900 !important;
```

Comment:
This is fixed in the toolkit PR, works fine now and should be merged soon! Good find